controlplane/kubeadm/main.go

```diff
@@ -92,6 +93,7 @@ var (
 func init() {
 	_ = clientgoscheme.AddToScheme(scheme)
 	_ = clusterv1.AddToScheme(scheme)
+	_ = expv1.AddToScheme(scheme)
```

Comment:
Wondering which change leads to requiring this import 🤔

Reply:
Exactly!